requirements.txt

```diff
@@ -1,3 +1,6 @@
 # This file is here because many Platforms as a Service look for
 #	requirements.txt in the root directory of a project.
 -r requirements/production.txt
+
+commercetools==14.0.0.0
+Pillow
```

Comment:
Requirements failed on my local, so I added these to requirements/base.in instead.